app/uk/gov/hmrc/apiplatform/modules/submissions/views/CredentialsRequestedView.scala.html

```diff
@@ -47,8 +47,32 @@ <h1 class="govuk-heading-xl">
 
     <div class="govuk-body">
       <p>We're checking the information you've provided and reviewing your testing.</p>
+      <p>The checking process can last at least 10 working days.</p>
```

Comment:
I know you didn't write this, but the wording seems odd 'can last' suggests it might not last this long, and 'at least' suggests it might last longer - so this sentence is saying it might take either more or less than 10 days.